_includes/components/button.html

```diff
@@ -0,0 +1,9 @@
+{% assign class = "button " | append: include.class %}
```

Comment:
Nice! I really like listing all the optional values we can pass in at the top - makes it super easy to read and document!